cfme/tests/migration_analytics/test_migration_analytics.py

```diff
@@ -0,0 +1,206 @@
+"""Tests around the Migration Analytics"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    pytest.mark.ignore_stream("5.10"),
+    pytest.mark.tier(1),
+    test_requirements.migration_analytics
+]
+
+
+@pytest.mark.manual
+@pytest.mark.smoke
+def test_enable_migration_analytics():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/4h
+        startsin: 5.11
+        casecomponent: MigrationAnalytics
+        testSteps:
+            1. Navigate to configuration > Zones > Zone: Default Zone > Server
+            2. Click on Advanced tab
+            3. Update script as below:
+               :prototype:
+                    :migration_analytics:
+                    :enabled: true
```

Comment:
Just a note, you can use `appliance.update_advanced_settings(data_dictionary)` to update this instead of all the navigation.

Reply:
Ok. Thanks for this suggestion. keeping these steps as it to understand test case.